sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -392,5 +394,92 @@ def test_global_fanout(self):
       assert_that(result, equal_to([49.5]))
 
 
+class LatestTest(unittest.TestCase):
+
+  def test_globally(self):
+    l = [window.GlobalWindows.windowed_value(1, 100),
```

Comment:
Maybe shuffle the example inputs a bit to make them look more random? Like [(2, 100), (3, 300), (1, 200)]? Same below.

Reply:
Sure! I'll make the change.